vms/vms/urls.py

```diff
@@ -5,6 +5,7 @@
 admin.autodiscover()
 
 urlpatterns = patterns('',
+    url(r'^$', include('home.urls', namespace='home')),
     url(r'^admin/', include(admin.site.urls)),
```

Comment:
Thank you.

I will revert the settings.py change and include only this and create a
different pull request.
Is that Ok ?



On Mon, Oct 19, 2015 at 7:15 AM, Tapasweni Pathak <notifications@github.com>
wrote:

> In vms/vms/urls.py
> <https://github.com/systers/vms/pull/155#discussion_r42363606>:
>
> > @@ -5,6 +5,7 @@
> >  admin.autodiscover()
> >
> >  urlpatterns = patterns('',
> > +    url(r'^$', include('home.urls', namespace='home')),
> >      url(r'^admin/', include(admin.site.urls)),
>
> This is great. [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/vms/pull/155/files#r42363606>.
>